ui/app/components/clients/page/overview.hbs

```diff
@@ -16,14 +16,12 @@
 {{#if this.hasAttributionData}}
   <Clients::Attribution
     @isSecretsSyncActivated={{this.flags.secretsSyncIsActivated}}
-    @newUsageCounts={{this.newClientCounts}}
     @totalClientAttribution={{this.totalClientAttribution}}
     @newClientAttribution={{this.newClientAttribution}}
     @selectedNamespace={{@namespace}}
     @startTimestamp={{@startTimestamp}}
     @endTimestamp={{@endTimestamp}}
     @responseTimestamp={{@activity.responseTimestamp}}
     @isHistoricalMonth={{and (not this.isCurrentMonth) (not this.isDateRange)}}
-    @upgradesDuringActivity={{this.upgradesDuringActivity}}
```

Comment:
since this is no longer being used I think we can remove the `upgradesDuringActivity` getter from the activity component. 

https://github.com/hashicorp/vault/blob/266ea693ccdec27718c17b7ee354cc6e19d9f305/ui/app/components/clients/activity.ts#L107-L110

Reply:
I checked and it's still being used in the runningTotals component